### DISCUSSION TITLE
Failed to resolve component: picker

### DISCUSSION THREAD

**Discussion by holyrong** [jdf2e/nutui#942] [Q&A]
版本：Taro3.4 NutUI3.1.14 typescript vue3
按照官网步骤，Taro init选择了  Vue3 + NutUI 模板
按需加载：
![image](https://user-images.githubusercontent.com/8318475/149126718-6e73d8d7-b27d-4dc4-ba99-3bcc43275510.png)

vue页面代码：
<nut-picker mode="multiSelector" :list-data="tenants" :default-value-index="currentIndex" @confirm="onChange" ></nut-picker>

vscode编译正常，小程序控制台报以下错误：
[Vue warn]: Failed to resolve component: picker
![image](https://user-images.githubusercontent.com/8318475/149126844-750a0395-d820-4cf8-a396-7c4f84755bb8.png)

**Comment by holyrong** [comment]
![image](https://user-images.githubusercontent.com/8318475/149127030-775b3ad9-9f94-46e3-8892-7ea0a748f73b.png)

**Comment by richard1015** [comment]
3.1.19-beta试试看，已修复
